python_modules/dagster/dagster/_core/definitions/declarative_scheduling/operands/parent_newer_condition.py

```diff
@@ -0,0 +1,106 @@
+from typing import AbstractSet
+
+from dagster._core.asset_graph_view.asset_graph_view import AssetSlice
+from dagster._core.definitions.declarative_scheduling.scheduling_condition import (
+    SchedulingCondition,
+    SchedulingResult,
+)
+from dagster._core.definitions.declarative_scheduling.scheduling_context import SchedulingContext
+from dagster._core.definitions.events import AssetKeyPartitionKey
+from dagster._serdes.serdes import whitelist_for_serdes
+
+
+@whitelist_for_serdes
+class ParentNewerCondition(SchedulingCondition):
+    @property
+    def description(self) -> str:
+        return "At least one parent has been updated more recently than the candidate."
+
+    def compute_newer_parents(
```

Comment:
what is the relationship between newer parents and our new language around "unsynced"?